src/test/java/bogdanSandu/tests/tema8/pageObjects/BasicAjaxPage.java

```diff
@@ -0,0 +1,60 @@
+package bogdanSandu.tests.tema8.pageObjects;
+
+import bogdanSandu.tests.tema8.tests.BaseTest;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.Select;
+import org.testng.Assert;
+
+public class BasicAjaxPage extends BaseTest {
+
+    private WebDriver driver;
+
+    public BasicAjaxPage(ChromeDriver driver){
+        this.driver = driver;
+        PageFactory.initElements(driver,this);
+    }
+
+    public static final String START_PAGE_URL = "https://testpages.herokuapp.com/styled/basic-ajax-test.html";
+
+    @FindBy(css ="#combo1")
+    private WebElement selectCategory;
+
+    @FindBy(css ="#combo2")
+    private WebElement selectLanguage;
+
+    @FindBy(css ="input.styled-click-button")
+    private WebElement button;
+
+    @FindBy(id ="_valueid")
+    private WebElement categorySelected;
+
+    @FindBy(id ="_valuelanguage_id")
+    private WebElement languageSelected ;
+
+    public void goToPage() {
+        driver.get(START_PAGE_URL);
+    }
+
+    public void clickOnButton(){
+        button.click();
+        fluentWait((ChromeDriver) driver).until(ExpectedConditions.visibilityOfElementLocated(By.id("_valueid")));
```

Comment:
Declare the and initialise the fluentWait either in this page constructor or in a separated WaitUtils class inside utils package.